net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -352,6 +352,46 @@ mwan3_set_custom_ipset()
 	error=$(echo "$update" | $IPS restore 2>&1) || LOG error "set_custom_ipset: $error"
 }
 
+mwan3_load_ipset_rule()
+{
+	local family
+	local rule=$1
+
+	config_get family $rule family 'any'
+	local dir ips
+	local has_ipv6=0
+	local has_ipv4=0
```

Comment:
what these vars(`has_ipv6/has_ipv4`) for?

Reply:
thank you. These are not needed.